nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -14,8 +14,12 @@
     <license type="file">LICENSE</license>
     <projectUrl>https://github.com/microsoft/cswinrt/tree/master/</projectUrl>
     <dependencies>
-      <group targetFramework=".NETStandard2.0" />
-      <group targetFramework=".NET5.0" />
+      <group targetFramework=".NETStandard2.0" >
+        <dependency id="Microsoft.Net.Compilers.Toolset" version="3.7.0" />
```

Comment:
I don't think we should add the compiler toolset package as a dependency. We don't want to force a downgrade for anyone building with a more recent preview (this package doesn't do downgrades well). And we're going to remove this dependency once .NET 5.0 releases.

Reply:
My main goal was to avoid users getting confused why it isn't building, but I agree this is a dependency that comes from VS and if they were using a later VS version, they would have a newer version of the package.  I updated the readme to communicate it.